src/models/m_identity.erl

```diff
@@ -358,9 +364,9 @@ check_username_pw("admin", Password, Context) ->
             {error, password}
     end;
 check_username_pw(Username, Password, Context) ->
-    Username1 = z_string:trim(z_string:to_lower(Username)),
+    Username1 = ?DEBUG(z_string:trim(z_string:to_lower(Username))),
```

Comment:
No, `mod_admin_identity` was running on my local machine for `testsandboxdb`. It was probably started. The schema of the test site should be dropped before running the tests.